src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.1.0
 -----
 
+ * Deprecated `Symfony\Component\Form\Extension\Validator\Util\ServerParams` in favor of its parent class `Symfony\Component\Form\Util\ServerParams`
```

Comment:
let's list it last, better let the headline to features

Reply:
Done